spinalcordtoolbox/deepseg/inference.py

```diff
@@ -59,7 +59,13 @@ def segment_and_average_volumes(model_paths, input_filenames, options):
         if len(model_paths) > 1:  # We have an ensemble, so output messages to distinguish between seeds
             name_seed = Path(path_model).parts[-2]
             logger.info(f"\nUsing '{name_seed}'...")
-        nii_lst, target_lst = imed_inference.segment_volume(path_model, input_filenames, options=options)
+        # NB: ivadomed turns on GPU inference via specifying a single GPU ID. This isn't the recommended way to do
+        #     things, since it prevents us from running multi-GPU jobs. I think ivadomed did things this way to limit
+        #     which GPU is used, but we can already accomplish this using the more universal 'CUDA_VISIBLE_DEVICES'
+        #     environment variable). For now, the best we can do on our end is to select the first GPU from the list
+        #     of available GPUs.
+        gpu_id = 0 if use_gpu else None  # NB: If e.g. 'CUDA_VISIBLE_DEVICES=2,3,4', then 0 will refer to GPU 2.
```

Comment:
Thank you for this comment! It will be very useful for whoever reads the code next.